src/nvim/api/keysets.lua

```diff
@@ -5,6 +5,9 @@ return {
   set_extmark = {
     "id";
     "end_line";
+    -- nvim_buf_get_extmark uses "end_row" rather than "end_line" in its
```

Comment:
I don't think we should add comments here, better to mention it inside the function if needed.